src/drivers/driver_zephyr.c

```diff
@@ -48,6 +49,21 @@ void wpa_supplicant_event_wrapper(void *ctx,
 				os_memcpy(ies, data->auth.ies, data->auth.ies_len);
 				data_tmp->auth.ies = ies;
 			}
+		} else if (event == EVENT_RX_MGMT) {
+			union wpa_event_data *data_tmp = msg.data;
+
+			if (data->rx_mgmt.frame) {
+				char *frame = os_zalloc(data->rx_mgmt.frame_len);
+
+				if (!frame) {
+					wpa_printf(MSG_ERROR, "%s: Failed to alloc frame\n",
+						__func__);
```

Comment:
Indentation seems to be off.

Reply:
For this and other indentation comments, it is expected for the indent to be off, for two reasons:

1. Using spaces to indent isn't allowed in checkpatch
2. Using tabs + spaces to align with open brace depends on tab->spaces conversion (intermixing of tabs and spaces), so, every editor (including GH diff) shows it differently, unless same conversion rules are followed.